### PR TITLE
Add validation of the certificate fields.

### PR DESCRIPTION
**Pull request by psantos1113**

### CODE REVIEW

File: alert-common/src/main/java/com/synopsys/integration/alert/common/security/CertificateUtility.java [hunk 1]
```diff
@@ -145,6 +145,10 @@ public File getAndValidateTrustStoreFile() throws AlertConfigurationException {
         }
     }
 
+    public void validateCertificateContent(CustomCertificateModel customCertificateModel) throws AlertException {
```
Comment (gkillough) [inline alert-common/src/main/java/com/synopsys/integration/alert/common/security/CertificateUtility.java]: This method and the method below this both take a customCertificateModel. How does a user of the API know which validate to call?

Comment (psantos1113) [reply alert-common/src/main/java/com/synopsys/integration/alert/common/security/CertificateUtility.java]: The method below is private so they wouldn't know it exists. I can change the public method to validateCertificateContents.  I am just validating that the content is a in fact a certificate we can parse and potentially load.

Comment (gkillough) [reply alert-common/src/main/java/com/synopsys/integration/alert/common/security/CertificateUtility.java]: Ah, I see. I would still prefer if there was more distinction between what the methods do.

Comment (psantos1113) [reply alert-common/src/main/java/com/synopsys/integration/alert/common/security/CertificateUtility.java]: I have a change I will push soon to better distinguish between the two.  It's a valid point.  Naming things is always the fun part.

**Pull request merged**